medicines/search-client/src/lib.rs

```diff
@@ -196,9 +196,6 @@ fn build_search(
             ("api-version", config.api_version.as_str()),
             ("highlight", "content"),
             ("queryType", "full"),
-            ("@count", "true"),
```

Comment:
is it better to fix them now? `$` it up?

Reply:
I _think_ I remember reading that including count, top and skip increases query complexity, which means we'll hit usage limits sooner. [The documentation is very vague about this though.](https://docs.microsoft.com/en-us/azure/search/search-limits-quotas-capacity#queries-per-second-qps)